lib/utils/fast-boot.js

```diff
@@ -0,0 +1,22 @@
+const { getParent } = require('../utils/common');
+
+function isEnvironmentBrowser(node) {
+  let callee = node.test.callee;
+
+  if (callee) {
+    let obj = callee.object;
+    let prop = callee.property;
+
+    return obj.name === 'environment' && prop.name === 'isBrowser';
```

Comment:
I am not in favor of this since this is in the open source and is leaking implementation details of something internal for us 😄 .